>From 2dac1e1583fa2d0f142a048e275845e829883777 Mon Sep 17 00:00:00 2001
From: Erik van Pienbroek <epienbro@fedoraproject.org>
Date: Sun, 19 Jan 2014 17:28:12 +0100
Subject: [PATCH 2/2] Remove rand_s and vsprintf_s from msvcrt.def.in

These symbols are implemented in the secapi wrappers so they
shouldn't be part of the msvcrt.def.in files any more
---
 mingw-w64-crt/lib32/msvcrt.def.in | 2 +-
 mingw-w64-crt/lib64/msvcrt.def.in | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/mingw-w64-crt/lib32/msvcrt.def.in b/mingw-w64-crt/lib32/msvcrt.def.in
index 6c2c3cf..3455447 100644
--- a/mingw-w64-crt/lib32/msvcrt.def.in
+++ b/mingw-w64-crt/lib32/msvcrt.def.in
@@ -1218,7 +1218,7 @@ vsnprintf == _vsnprintf
 snprintf == _snprintf
 snwprintf == _snwprintf
 vsnwprintf == _vsnwprintf
-vsprintf_s
+; vsprintf_s replaced by emu
 vswprintf_s
 vwprintf_s
 wcrtomb_s
diff --git a/mingw-w64-crt/lib64/msvcrt.def.in b/mingw-w64-crt/lib64/msvcrt.def.in
index a0d3647..ceff01e 100644
--- a/mingw-w64-crt/lib64/msvcrt.def.in
+++ b/mingw-w64-crt/lib64/msvcrt.def.in
@@ -1160,7 +1160,7 @@ qsort
 qsort_s
 raise
 rand
-rand_s
+; rand_s replaced by emu
 realloc
 remove
 rename
@@ -1239,7 +1239,7 @@ vprintf_s
 vsnprintf == _vsnprintf
 snprintf == _snprintf
 vsprintf
-vsprintf_s
+; vsprintf_s replaced by emu
 vswprintf
 vswprintf_s
 vwprintf
-- 
1.8.4.2

